<a href="https://colab.research.google.com/github/CM-007/DS_ASS/blob/main/Ass01Q7.ipynb" target="_parent"><img src="https://colab.research.google.com/assets/colab-badge.svg" alt="Open In Colab"/></a>

Q7) Calculate Mean, Median, Mode, Variance, Standard Deviation, Range &     comment about the values / draw inferences, for the given dataset

-	For Points,Score,Weigh>
Find Mean, Median, Mode, Variance, Standard Deviation, and Range and also Comment about the values/ Draw some inferences.


# **Import libraries**

In [1]:
import pandas as pd

In [ ]:
data = pd.read_csv('/content/Q7.csv')

In [4]:
data.describe()

Points      Score      Weigh
count  32.000000  32.000000  32.000000
mean    3.596563   3.217250  17.848750
std     0.534679   0.978457   1.786943
min     2.760000   1.513000  14.500000
25%     3.080000   2.581250  16.892500
50%     3.695000   3.325000  17.710000
75%     3.920000   3.610000  18.900000
max     4.930000   5.424000  22.900000

This is a car data set with columns Points,Score and Weight.
* In this data set no null value present.
* count is a 32 rows and 3 columns.

* Mean - is avrage of data
* Median - is a mi
* mode - most frequently occurring value
* Variance - mean of squared deviation score
* Standard Deviation -is square rootv of a variance
* range - is a diffrence between the largest value and smallest value



In [5]:
data.head()

Unnamed: 0  Points  Score  Weigh
0          Mazda RX4    3.90  2.620  16.46
1      Mazda RX4 Wag    3.90  2.875  17.02
2         Datsun 710    3.85  2.320  18.61
3     Hornet 4 Drive    3.08  3.215  19.44
4  Hornet Sportabout    3.15  3.440  17.02

In [26]:
print("Mean :",data['Points'].mean())
print("Median :",data['Points'].median())
print("Mode :",data['Points'].mode())
print("Variance :",data['Points'].var())
print("Standard Deviation :",data['Points'].std())
print("Range :",data.Points.max() - data.Points.min())

Mean : 3.5965625
Median : 3.6950000000000003
Mode : 0    3.07
1    3.92
Name: Points, dtype: float64
Variance : 0.2858813508064516
Standard Deviation : 0.5346787360709715
Range : 2.17


In points column multi model data occurce
and range is 2.17

In [27]:
print("Mean :",data['Score'].mean())
print("Median :",data['Score'].median())
print("Mode :",data['Score'].mode())
print("Variance :",data['Score'].var())
print("Standard Deviation :",data['Score'].std())
print("Range :",data.Score.max() - data.Score.min())

Mean : 3.2172500000000004
Median : 3.325
Mode : 0    3.44
Name: Score, dtype: float64
Variance : 0.9573789677419354
Standard Deviation : 0.9784574429896966
Range : 3.9110000000000005


In points column Bi model data occurce
and range is 3.91

In [28]:
print("Mean :",data['Weigh'].mean())
print("Median :",data['Weigh'].median())
print("Mode :",data['Weigh'].mode())
print("Variance :",data['Weigh'].var())
print("Standard Deviation :",data['Weigh'].std())
print("Range :",data.Weigh.max() - data.Weigh.min())

Mean : 17.848750000000003
Median : 17.71
Mode : 0    17.02
1    18.90
Name: Weigh, dtype: float64
Variance : 3.193166129032258
Standard Deviation : 1.7869432360968431
Range : 8.399999999999999


In [ ]:
print("Mean :",data['Weigh'].mean())
print("Median :",data['Weigh'].median())
print("Mode :",data['Weigh'].mode())
print("Variance :",data['Weigh'].var())
print("Standard Deviation :",data['Weigh'].std())
print("Range :",data.Weigh.max() - data.Weigh.min())

Mean : 17.848750000000003
Median : 17.71
Mode : 0    17.02
1    18.90
Name: Weigh, dtype: float64
Variance : 3.193166129032258
Standard Deviation : 1.7869432360968431
Range : 8.399999999999999


In points column multi model data occurce and range is 8.3